mappings/net/minecraft/network/packet/s2c/play/BossBarS2CPacket.mapping

```diff
@@ -1,3 +1,126 @@
+			ARG 1 uuid
+			ARG 2 color
+			ARG 3 style
+		METHOD method_34102 handleUpdateName (Ljava/util/UUID;Lnet/minecraft/class_2561;)V
+			ARG 1 uuid
+			ARG 2 name
+		METHOD method_34103 handleAdd (Ljava/util/UUID;Lnet/minecraft/class_2561;FLnet/minecraft/class_1259$class_1260;Lnet/minecraft/class_1259$class_1261;ZZZ)V
+			ARG 1 uuid
+			ARG 2 name
+			ARG 3 percent
+			ARG 4 color
+			ARG 5 style
+			ARG 6 darkenSky
+			ARG 7 dragonMusic
+			ARG 8 thickenFog
+		METHOD method_34104 handleUpdateProperties (Ljava/util/UUID;ZZZ)V
+			ARG 1 uuid
+			ARG 2 darkenSky
+			ARG 3 dragonMusic
+			ARG 4 thickenFog
+	CLASS class_5882 BossBarAction
+		METHOD method_34105 getType ()Lnet/minecraft/class_2629$class_5883;
+		METHOD method_34106 handle (Ljava/util/UUID;Lnet/minecraft/class_2629$class_5881;)V
+			ARG 1 uuid
+			ARG 2 handler
+		METHOD method_34107 write (Lnet/minecraft/class_2540;)V
+			ARG 1 buf
+	CLASS class_5883 Type
+		FIELD field_29107 ADD Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29108 REMOVE Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29109 UPDATE_PROGRESS Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29110 UPDATE_NAME Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29111 UPDATE_STYLE Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29112 UPDATE_PROPERTIES Lnet/minecraft/class_2629$class_5883;
+		FIELD field_29113 typeFactory Ljava/util/function/Function;
+		METHOD <init> (Ljava/lang/String;ILjava/util/function/Function;)V
+			ARG 3 typeFactory
+		METHOD method_34108 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+		METHOD method_34110 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+		METHOD method_34111 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+		METHOD method_34112 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+		METHOD method_34113 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+		METHOD method_34114 (Lnet/minecraft/class_2540;)Lnet/minecraft/class_2629$class_5882;
+			ARG 0 buf
+	CLASS class_5884 UpdateName
+		FIELD field_29115 name Lnet/minecraft/class_2561;
+		METHOD <init> (Lnet/minecraft/class_2540;)V
+			ARG 1 buf
+		METHOD <init> (Lnet/minecraft/class_2561;)V
+			ARG 1 name
+	CLASS class_5885 UpdateProgress
+		FIELD field_29116 percent F
+		METHOD <init> (F)V
+			ARG 1 percent
+		METHOD <init> (Lnet/minecraft/class_2540;)V
+			ARG 1 buf
+	CLASS class_5886 UpdateProperties
+		FIELD field_29117 darkenSky Z
+		FIELD field_29118 dragonMusic Z
+		FIELD field_29119 thickenFog Z
+		METHOD <init> (Lnet/minecraft/class_2540;)V
+			ARG 1 buf
+		METHOD <init> (ZZZ)V
+			ARG 1 darkenSky
+			ARG 2 dragonMusic
+			ARG 3 thickenFog
+	CLASS class_5887 UpdateStyle
+		FIELD field_29120 color Lnet/minecraft/class_1259$class_1260;
+		FIELD field_29121 style Lnet/minecraft/class_1259$class_1261;
```

Comment:
Should be `overlay`
https://github.com/FabricMC/yarn/commit/1bc0fa389c4f42e0627a6c629852eda28239e2f1#diff-d1e3850a3e0b082f516b761f59c42f96b707a305c97913c56bb57a6884097bacL10

Reply:
Class name is `Style`, it really should be `style` rather than `overlay`, i just forgot to change it in `Add` class. Whats better?

Reply:
Oh, i see. It is for whatever reason indeed called `getOverlay` in `BossBar`.

Reply:
Hmm, it's indeed weird. In `BossBar` the getter/setter is `overlay` but field is `style`